<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PassGen</class>
 <widget class="QMainWindow" name="PassGen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PassGen</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_special">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>500</y>
      <width>150</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn_special:hover {
	border: none;
	background-color: rgb(245, 245, 245);
}
#btn_special:checked {
	border: none;
	background-color: rgb(165, 255, 123);
}</string>
    </property>
    <property name="text">
     <string>1O0Il</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_gen">
    <property name="geometry">
     <rect>
      <x>149</x>
      <y>500</y>
      <width>251</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn_gen {
	border: none;
	background-color: rgb(96, 209, 62);
}
#btn_gen:hover {
	border: none;
	background-color: rgb(169, 211, 160);
}
#btn_gen:pressed {
	border: none;
	background-color: rgb(165, 255, 123);
}</string>
    </property>
    <property name="text">
     <string>GEN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_symbols">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>400</y>
      <width>150</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn_symbols:hover {
	border: none;
	background-color: rgb(245, 245, 245);
}
#btn_symbols:checked {
	border: none;
	background-color: rgb(165, 255, 123);
}</string>
    </property>
    <property name="text">
     <string>#%$</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_abc">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>300</y>
      <width>150</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn_abc:hover {
	border: none;
	background-color: rgb(245, 245, 245);
}
#btn_abc:checked {
	border: none;
	background-color: rgb(165, 255, 123);
}</string>
    </property>
    <property name="text">
     <string>abc</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ABC">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>150</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn_ABC:hover {
	border: none;
	background-color: rgb(245, 245, 245);
}
#btn_ABC:checked {
	border: none;
	background-color: rgb(165, 255, 123);
}</string>
    </property>
    <property name="text">
     <string>ABC</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_123">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>150</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn_123:hover {
	border: none;
	background-color: rgb(245, 245, 245);
}
#btn_123:checked {
	border: none;
	background-color: rgb(165, 255, 123);
}</string>
    </property>
    <property name="text">
     <string>123</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_save">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>300</y>
      <width>150</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn_save:hover {
	border: none;
	background-color: rgb(245, 245, 245);
}
#btn_save:pressed {
	border: none;
	background-color: rgb(165, 255, 123);
}</string>
    </property>
    <property name="text">
     <string>SAVE</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_login">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>400</y>
      <width>150</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn_login:hover {
	border: none;
	background-color: rgb(245, 245, 245);
}
#btn_login:pressed {
	border: none;
	background-color: rgb(165, 255, 123);
}</string>
    </property>
    <property name="text">
     <string>LOGIN</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_find">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>200</y>
      <width>150</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn_open:hover {
	border: none;
	background-color: rgb(245, 245, 245);
}
#btn_open:pressed {
	border: none;
	background-color: rgb(165, 255, 123);
}</string>
    </property>
    <property name="text">
     <string>FIND</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_enter">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>150</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn_enter {
	border: none;
	background-color: rgb(187, 223, 255);
}

#btn_enter:hover {
	border: none;
	background-color: rgb(169, 212, 240);
}
#btn_enter:pressed {
	border: none;
	background-color: rgb(165, 255, 123);
}
</string>
    </property>
    <property name="text">
     <string>ENT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_add_symbols">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>400</y>
      <width>101</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn_add_symbols:hover {
	border: none;
	background-color: rgb(245, 245, 245);
}
#btn_add_symbols:checked {
	border: none;
	background-color: rgb(165, 255, 123);
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_add_abc">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>300</y>
      <width>101</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn_add_abc:hover {
	border: none;
	background-color: rgb(245, 245, 245);
}
#btn_add_abc:checked {
	border: none;
	background-color: rgb(165, 255, 123);
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_add_ABC">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>101</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn_add_ABC:hover {
	border: none;
	background-color: rgb(245, 245, 245);
}
#btn_add_ABC:checked {
	border: none;
	background-color: rgb(165, 255, 123);
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_add_123">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>101</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn_add_123:hover {
	border: none;
	background-color: rgb(245, 245, 245);
}
#btn_add_123:checked {
	border: none;
	background-color: rgb(165, 255, 123);
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>39</y>
      <width>401</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>28</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 238, 238);</string>
    </property>
    <property name="text">
     <string>Укажите длину пароля и нажмите ENT</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="margin">
     <number>5</number>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
